src/Router/RouteFactoryInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Equip\Queue\Router;
+
+use Equip\Queue\Exception\HandlerException;
+use Equip\Queue\Exception\RouterException;
+
+interface RouteFactoryInterface
+{
+    /**
+     * Retrieve callable for message
```

Comment:
Based on this comment, it seems like this interface should be named `HandlerFactoryInterface`?